eclipse-che/che-server
# Align the unreachable devfile exception message

### What does this PR do?
Add a common exception message and use it for cases when:
* The devfile location is unavailable.
* The devfile content is is not valid: e.g the response content is an html content of an scm authorisation page.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/22469

### How to test this PR?

1. Deploy che from the PR image: `quay.io/eclipse/che-server:pr-615`
2. Start a workspace from a private GitHub repo, see the error:
![screenshot-192 168 64 214 nip io-2023 11 20-15_17_57](https://github.com/eclipse-che/che-server/assets/7668752/ff34a518-d24c-475d-91ce-e658f3a2748d)
3. Start a workspace from a private azure-devops repo, see the error in the same format:
![screenshot-192 168 64 214 nip io-2023 11 20-15_18_37](https://github.com/eclipse-che/che-server/assets/7668752/9a44dbd6-868b-458f-b726-cc21904a5de4)




### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
